pkg/graph/resolvers/governance_request_feedback.go

```diff
@@ -28,7 +28,7 @@ func GetGovernanceRequestFeedbackAuthor(
 	}
 
 	// Handled nil value above, safe to dereference and pass to GetUserInfo
-	authorInfo, err := dataloaders.GetUserInfo(ctx, *feedbackAuthorEUAID)
+	authorInfo, err := dataloaders.FetchUserInfosByEUAUserID(ctx, *feedbackAuthorEUAID)
```

Comment:
I know this might just be conflicting naming, but it's a bit confusing here to read `FetchUserInfos` (namely, the `s`) when we're really only fetching a _single_ UserInfo

Perhaps we should opt to name all dataloaders in the singular, and leave the `batch` functions to handle the plural?

Reply:
you are absolutely correct - great call! Nice catch